app-games/src/main/java/com/aptoide/android/aptoidegames/gamegenie/presentation/GameGenieFeatureProvider.kt

```diff
@@ -36,3 +44,25 @@ fun rememberGameGenieVisibility(): Boolean = runPreviewable(
     state
   }
 )
+
+@Composable
+fun rememberGameGenieApp(app: App): Pair<AppUiState, () -> Unit> = runPreviewable(
```

Comment:
as said before is this needed ? it will only emmit the app that we already had, nothing else